### PR TITLE
[WIP]Makes all global vars use the GLOB define.

### PR DESCRIPTION
**Pull request by GrayRachnid**
Words cannot describe my suffering.

### CODE REVIEW

File: code/New_Hud_Manual_ru.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //������������ �������: flash, pain, blind, damageoverlay, blurry, druggy, nvg, thermal, meson, science
 ������������ ���������� ����������:
-global.HUDdatums - ��� �������� ������ � �������� /datum/hud, ������������ ���������� name ������ = ��� �����
+global.GLOB.HUDdatums - ��� �������� ������ � �������� /datum/hud, ������������ ���������� name ������ = ��� �����
```
Comment (mattroks101) [inline code/New_Hud_Manual_ru.dm]: That seems like a typo.

**Comment by GrayRachnid** [comment]
i should finish this

**Comment by GrayRachnid** [comment]
this is now up for grabs i'm too busy

**Comment by SpadesSlick** [comment]
Staled at this point.

**Pull request closed without merging**